### ISSUE TITLE
documentation page for a package should list the libraries in it frst

### ISSUE THREAD

**Issue by skybrian** [jsr-io/jsr#606]
I'm looking at:

https://jsr.io/@std/testing/doc

The asserts library is listed first, and is mostly deprecated. It would be nice to be able to collapse it.

Also, perhaps large packages should start with a list of libraries within them, as an overview?

Compare with the [same page on deno.land](https://deno.land/std@0.224.0/testing), which does have an index. (And it's all that's there.)

**Comment by skybrian** [comment]
To add some context, these groupings are "entrypoints," which are documented here: https://jsr.io/docs/using-packages#entrypoints

**Comment by dgreensp** [comment]
Yes, my first experience with JSR was being confused by the fact that the entrypoints of a package don't seem to be listed anywhere, though you can read deno.json in the Files tab.  In actuality, they are the section names in the Docs tab, but there is no table of contents.

An issue that added to the confusion for me is that the "default" entrypoint does not seem to get a Docs page that lists its symbols.  If you click the word "default," for example on https://jsr.io/@std/io/doc, it just takes you to the Overview tab.  (It also takes two taps to click "default" on mobile, strangely.)  Because it's not clear what "default" refers to, and clicking on it does not take you to a Docs page, and the section names don't look like entrypoints (or anything in particular), coupled with the lack of TOC, the whole thing is disorienting and doesn't make it seem like you are being shown the modules within a package.

**Comment by dgreensp** [comment]
I know this issue might sit for a while, but please consider improving the Docs navigation.

While I didn't consciously know what the problem was initially, the "lack of a sense of place" and lack of navigation on JSR made me not want to use it initially.

Having the docs is one of the selling points of JSR, but the fact that they are not really good docs (compared to other docs site generators, no offense) creates a bit of a dilemma.
